pypgatk/commands/dnaseq_to_proteindb.py

```diff
@@ -9,8 +9,8 @@
 
 
 @click.command("dnaseq-to-proteindb", short_help="Generate peptides based on DNA sequences")
-@click.option('--config_file', '-c', help='Configuration to perform conversion between ENSEMBL Files',
-              default= this_dir + '../config/ensembl_config.yaml')
+@click.option('-c', '--config_file', help='Configuration to perform conversion between ENSEMBL Files',
+              default= this_dir + '/../config/ensembl_config.yaml')
```

Comment:
using `this_dir` makes issues when the default is used. why not just the default config file without `this_dir`?